### PR TITLE
docs: add explicit provider registration of new services in prisma recipe

### PR DESCRIPTION
**Pull request by pablopaul**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/recipes/prisma.md [hunk 1]
```diff
@@ -391,6 +391,21 @@ export class PostService {
 
 Your `UserService` and `PostService` currently wrap the CRUD queries that are available in Prisma Client. In a real world application, the service would also be the place to add business logic to your application. For example, you could have a method called `updatePassword` inside the `UserService` that would be responsible for updating the password of a user.
 
+We need to register the new services with Nest so that it can perform the injection. We do this by editing our module file (`app.module.ts`) and adding the new services to the `providers` array of the `@Module()` decorator.
+
+```typescript
```
Comment (micalevisk) [inline content/recipes/prisma.md:396]: I'm not sure if we need to show any code for this here as registering providers is pretty standard in nestjs apps

Comment (pablopaul) [reply content/recipes/prisma.md:396]: @micalevisk I followed along the recipe, tried to run the app and ran into the error. What do you think about just mentioning it and leaving out the code?

Comment (micalevisk) [reply content/recipes/prisma.md:396]: yeah we can have that mention to recall the dev, but the code itself is redundant here

Comment (pablopaul) [reply content/recipes/prisma.md]: I adapted the change accordingly.

**Review by micalevisk — APPROVED** [review]

**Pull request merged**